apps/web/src/views/V3Info/hooks/index.ts

```diff
@@ -65,7 +65,7 @@ export const useProtocolData = (): ProtocolData | undefined => {
   const chainId = multiChainId[chainName]
   const { data } = useQuery({
     queryKey: [`v3/info/protocol/ProtocolData/${chainId}`, chainId],
-    queryFn: () => fetchProtocolData(chainIdToExplorerInfoChainName[chainId]),
+    queryFn: ({ signal }) => fetchProtocolData(chainIdToExplorerInfoChainName[chainId], signal),
     enabled: Boolean(chainId),
```

Comment:
Can switch to use explorer chain name query
And check enabled here
It can skip fetch for init defaulr bsc fetch